dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,34 @@
+package io.dropwizard.auth;
```

Comment:
Hello from techdebt analyzer
<!-- lookout footnote separator -->
_Comment made by the analyzer techdebt._